SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you found a security vulnerability in NetsPresso Trainer, please report issues at contact@nota.ai. We will check you report closely and give our best to fix the problem. When you have a security vulnerability to report, please do not open github issue or pull request so that everyone can notice.
```

Comment:
@illian01 ,
plz change the email to `netspresso@nota.ai` (decided by internal discussion)